### ISSUE TITLE
[Discussion] Working with some stores doesn't feel comfortable

### ISSUE THREAD

**Issue by CKGrafico** [ktsn/vuex-class#21]
Maybe I'm not using the best practices but I'm not confortable when I do things like..

```
import * as pageSettingsStore from '~/store/modules/page-settings';
import * as classGroupsStore from '~/store/modules/class-groups';
import * as userStore from '~/store/modules/user';
import * as studentsStore from '~/store/modules/students';
import * as teachersStore from '~/store/modules/teachers';

...

const PageSettingsState = namespace(pageSettingsStore.name, State);
const PageSettingsGetter = namespace(pageSettingsStore.name, Getter);
const ClassGroupsAction = namespace(classGroupsStore.name, Action);
const ClassGroupsGetter = namespace(classGroupsStore.name, Getter);
const ClassGroupsState = namespace(classGroupsStore.name, State);
const UserGetter = namespace(userStore.name, Getter);
const StudentsState = namespace(studentsStore.name, State);
const TeachersState = namespace(teachersStore.name, State);

...

export default class extends Vue {
    public isLoading = false;
    public editMode = false;

    @PageSettingsGetter singlePageSettings;
    @PageSettingsState settingsPages;
    @ClassGroupsState groups;

    @ClassGroupsGetter studentsInClass;
    @ClassGroupsGetter isStudentInClass;
    @UserGetter currentCourses;

    @ClassGroupsAction updateClassGroups;
    @ClassGroupsAction removeClassGroup;

    @StudentsState('list') students;
    @TeachersState('list') teachers;

....
```

**Comment by ktsn** [comment]
Maybe we should simplify `namespace` helper like following:

```js
const pageSettings = namespace(pageSettingStore.name)
const classGroups = namespace(classGroupsStore.name)
const user = namespace(userStore.name)
const students = namespace(studentsStore.name)
const teachers = namespace(teachersStore.name)

@Component
export default class extends Vue {
    public isLoading = false;
    public editMode = false;

    @pageSettings.Getter singlePageSettings;
    @pageSettings.State settingsPages;
    @classGroups.State groups;

    @classGroups.Getter studentsInClass;
    @classGroups.Getter isStudentInClass;
    @user.Getter currentCourses;

    @classGroups.Action updateClassGroups;
    @classGroups.Action removeClassGroup;

    @students.State('list') students;
    @teachers.State('list') teachers;
}
```

**Comment by CKGrafico** [comment]
Looks nice!

**Comment by ktsn** [comment]
Shipped on v0.3.1

**Comment by CKGrafico** [comment]
After some days using it I can confirm that is working really well :)
